third_party/libpg_query/grammar/statements/create_function.y

```diff
@@ -4,20 +4,49 @@
  *
  *****************************************************************************/
  CreateFunctionStmt:
-			CREATE_P macro_alias qualified_name param_list AS a_expr
-				{
-					PGCreateFunctionStmt *n = makeNode(PGCreateFunctionStmt);
-					n->name = $3;
-					n->params = $4;
-					n->function = $6;
-					$$ = (PGNode *)n;
-				}
+
+		CREATE_P macro_alias TABLE qualified_name param_list AS SelectStmt
```

Comment:
`CREATE TABLE MACRO` and `CREATE SCALAR MACRO` seems more consistent with the SQL syntax in general (e.g. it is `CREATE TEMPORARY TABLE`, not `CREATE TABLE TEMPORARY`).

Reply:
Yes I tried in the pg Parser 
>**CREATE TABLE MACRO**

But was getting serious shift reduce conflicts from insert.y  which I don't fully understand.